HousingSearchListener.Tests/V1/E2ETests/Stories/AddPersonToTenureTests.cs

```diff
@@ -66,7 +66,6 @@ public void PersonNotFound()
                 .And(g => _personApiFixture.GivenThePersonDoesNotExist(_tenureApiFixture.AddedPersonId))
                 .When(w => _steps.WhenTheFunctionIsTriggered(tenureId, _tenureApiFixture.MessageEventData, EventTypes.PersonAddedToTenureEvent))
                 .Then(t => _steps.ThenTheCorrelationIdWasUsedInTheApiCall(_tenureApiFixture.ReceivedCorrelationIds))
-                .Then(t => _steps.ThenTheCorrelationIdWasUsedInTheApiCall(_personApiFixture.ReceivedCorrelationIds))
```

Comment:
Why is this step removed?

Reply:
There was issue in other palce, already rolled back this change